nimbus/src/lib.rs

```diff
@@ -35,27 +36,27 @@ pub struct NimbusClient {
     http_client: Client,
     available_randomization_units: AvailableRandomizationUnits,
     app_context: AppContext,
-    db: Database,
-    nimbus_id: Uuid,
```

Comment:
`nimbus_id` should be cached by LLDB anyway, so I dropped our own cache of it in favor of simplicity.